modules/metering-reports.adoc

```diff
@@ -262,6 +262,37 @@ spec:
   reportingEnd: "2019-07-31T00:00:00Z"
 ----
 
+[id="metering-expiration_{context}"]
+== expiration
+
+Add the `expiration` field to set a retention period on a scheduled metering Report. You can avoid manually removing the Report by setting the `expiration` duration value. The retention period is equal to the Report `creationDate` plus the `expiration` duration. The Report is removed from the cluster at the end of the retention period if no other Reports or ReportQueries depend on the expiring Report.
+
+[NOTE]
+====
+Setting the `expiration` field is not recommended for roll-up or aggregated reports. If a Report is depended upon by other Reports or ReportQueries, then the Report is not removed at the end of the retention period. You can view the `report-operator` logs at debug level for the timing output around a Report retention decision.
+====
+
+For example, the following scheduled Report is deleted 30 minutes after the `creationDate` of the Report:
+[source,yaml]
+----
+apiVersion: metering.openshift.io/v1
+kind: Report
+metadata:
+  name: pod-cpu-request-hourly
+spec:
+  query: "pod-cpu-request"
+  schedule:
+    period: "weekly"
+  reportingStart: "2020-09-01T00:00:00Z"
+  expiration: "30m" <1>
+----
+<1> Valid time units for the `expiration` duration are `ns`, `us` (or `µs`), `ms`, `s`, `m`, and `h`.
```

Comment:
@ahardin-rh @bentito @pruan-rht Can you PTAL at these small updates? Thank you!

I added a callout for valid time units for the `expiration` field and the new [NOTE] (below) regarding timing expectations for the retention period.

Reply:
I think we should mention that it's likely to take longer than a minute for actual deletion to occur. The expiration end to end tests wait up 4 minutes on a 30s expiration to avoid having a flaky test for instance.